spec/ReorderProcessing/ReorderItemsProcessorSpec.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\CustomerReorderPlugin\ReorderProcessing;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Core\Model\OrderItemInterface;
+use Sylius\Component\Core\Model\ProductVariantInterface;
+use Sylius\Component\Inventory\Checker\AvailabilityCheckerInterface;
+use Sylius\Component\Order\Modifier\OrderItemQuantityModifierInterface;
+use Sylius\Component\Order\Modifier\OrderModifierInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Sylius\CustomerReorderPlugin\ReorderProcessing\ReorderItemsProcessor;
+use Sylius\CustomerReorderPlugin\ReorderProcessing\ReorderProcessor;
+
+final class ReorderItemsProcessorSpec extends ObjectBehavior
+{
+    function let(
+        OrderItemQuantityModifierInterface $orderItemQuantityModifier,
+        OrderModifierInterface $orderModifier,
+        AvailabilityCheckerInterface $availabilityChecker,
+        FactoryInterface $orderItemFactory
+    ) {
+        $this->beConstructedWith($orderItemQuantityModifier, $orderModifier, $availabilityChecker, $orderItemFactory);
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType(ReorderItemsProcessor::class);
+    }
+
+    function it_implements_reorder_processor_interface()
+    {
+        $this->shouldImplement(ReorderProcessor::class);
+    }
+
+    function it_copies_order_items_to_reorder(
+        FactoryInterface $orderItemFactory,
```

Comment:
As in previous PR, please, try to keep the same order of parameters as in `let` method